.github/workflows/ci.yml

```diff
@@ -38,8 +38,9 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '3.1'
-          - '3.0'
+          - 3.2
+          - 3.1
+          - 3.0
```

Comment:
This will be parsed as `3` and will become `3.2`: https://github.com/ruby/setup-ruby#matrix-of-ruby-versions